opensrp-reveal/src/main/assets/json.form/case_confirmation.json

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
In general, this form needs to be linked to the location.id, the family.id and the client.id so we can associate subsequent actions and appropriately report across it. I don't know how to do that.

Reply:
Also, this form does not have calculation rules. Please verify that we don't need calculation rules.

Reply:
we need calculation rules

Reply:
> In general, this form needs to be linked to the location.id, the family.id and the client.id so we can associate subsequent actions and appropriately report across it. I don't know how to do that.

These shall be loaded at runtime when opening the form, the location, task and family details will be populated into a details object